hub-chart/values-cp-only.yaml

```diff
@@ -0,0 +1,6 @@
+# KubeFlex override values

```

Comment:
Nothing references this file. What is it for?

Reply:
Do you mean that you want this file to NOT be in git _and_ you want it to be in `.gitignore`? You could do that --- IF you do it in two steps: first one commit that deletes the file, and then a commit that adds it to `.gitignore`. Does not have to be done in this PR.

Reply:
@MikeSpreitzer somehow I messed up my response above, the .gitignore response vas actually meant for the `Chart.lock` file.

This file was created because I was thinking of replacing this behavior:

```shell
helm upgrade --install add-wds2 oci://ghcr.io/kubestellar/kubestellar/hub-chart --version $KUBESTELLAR_VERSION \
  --set='kubeflex-operator.install=false,InstallPCHs=false' \
  --set-json='WDSes=[{name":"wds2"}]'
```

with this behavior using the file:

```shell
helm upgrade --install add-wds2 oci://ghcr.io/kubestellar/kubestellar/hub-chart --version $KUBESTELLAR_VERSION \
  --values=values-cp-only.yaml \
  --set-json='WDSes=[{name":"wds2"}]'
```

but I think I will delete the file for the moment.